python-lib/dku_model_fairness_report/fairness_metric_report.py

```diff
@@ -12,16 +12,22 @@
 class ModelFairnessMetric(object):
 
     @staticmethod
-    def get_available_metrics():
+    def get_available_metric_names():
         return [ModelFairnessMetric.demographic_parity.__name__,
                 ModelFairnessMetric.equality_of_opportunity.__name__,
                 ModelFairnessMetric.equalized_odds.__name__,
                 ModelFairnessMetric.predictive_rate_parity.__name__]
 
     @staticmethod
-    def _compute_confusion_matrix_metrics(y_true, y_pred, advantageous_outcome, sample_weight=None):
-        label_list = [x for x in np.unique(y_true) if x != advantageous_outcome]
-        label_list.append(advantageous_outcome)
+    def get_available_metric_functions():
+        return [ModelFairnessMetric.demographic_parity,
+                ModelFairnessMetric.equality_of_opportunity,
+                ModelFairnessMetric.equalized_odds,
+                ModelFairnessMetric.predictive_rate_parity
+            ]
+
+    @staticmethod
+    def _compute_confusion_matrix_metrics(y_true, y_pred, label_list, sample_weight=None):
         conf_matrix = confusion_matrix(y_true, y_pred, labels=label_list, sample_weight=sample_weight)
 
```

Comment:
You could have used ```confusion_matrix([0, 1, 0, 1], [1, 1, 1, 0]).ravel()``` as in the scikit example

Reply:
I don't understand your comment

Reply:
never mind, I think there is a prebuilt function that you override the confusion_matrix function